tasks/add_new_prebid_partner.py

```diff
@@ -116,6 +120,7 @@ def _get_value_id_from_cache(self, value_name):
       if value_obj['name'] == value_name:
         val_id = value_obj['id']
         break
+    logger.info(u'Value: {value} matches ID "{val_id}"'.format(value=value_name,val_id=val_id))
```

Comment:
Do you think this log info provides value to the typical user? It doesn't seem like it to me, so let's remove it.